draft/Contest_API.md

```diff
@@ -878,7 +870,7 @@ JSON elements of problem objects:
 | uuid              | string  | no        | yes       | UUID of the problem, as defined in the problem package.
 | label             | string  | yes       | no        | Label of the problem on the scoreboard, typically a single capitalized letter.
 | name              | string  | yes       | no        | Name of the problem.
-| ordinal           | ORDINAL | yes       | no        | Ordering of problems on the scoreboard.
+| ordinal           | number  | yes       | no        | A unique number useful only to order the problems, e.g. for the scoreboard.
```

Comment:
I would rephrase this as `A unique number that determines the order the problems, e.g. on the scoreboard.`

Reply:
Agreed, updated.